frontend/src/pages/DegreeWizard/YearStep/YearStep.tsx

```diff
@@ -37,7 +38,7 @@ const YearStep = ({ incrementStep }: Props) => {
             }}
             onChange={(_, [startYear, endYear]: [string, string]) => {
               const numYears = parseInt(endYear, 10) - parseInt(startYear, 10) + 1;
-              dispatch(updateDegreeLength(numYears));
+              axios.put('/user/updateDegreeLength', { numYears });
```

Comment:
need to grab the token and pass it in too

Reply:
Fixed